gen2_examples/04_rgb_encoding.py

```diff
@@ -26,7 +26,7 @@
 device.startPipeline()
 
 # Output queue will be used to get the encoded data from the output defined above
-q = device.getOutputQueue(name="h265")
+q = device.getOutputQueue(name="h265", maxSize=4, blocking=False)
```

Comment:
Better to keep `blocking=True` for the video encoder queues, to not have lost encoded packets -> glitches in the video? And bigger queue size, as those packets are usually small?

 Also in the other examples.